lib/genres.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'text_styles.dart';
+import 'movie.dart';
+
+class WidgetedGenres extends StatelessWidget {
+  final Movie movie;
+
+  const WidgetedGenres(
+    this.movie, {
+    super.key,
+  });
+
+  static const genresWordPadding = EdgeInsets.only(left: 20);
```

Comment:
This const is not necessary, you need to create a const only for the padding left value not for all the EdgeInsets

Reply:
Done